src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -269,85 +274,90 @@ public boolean shipInBounds(Coordinate c, int orientation, int length){
     }
 
 
-
+/*Checks if ship placement at specified location would be inbounds, not overlap any other ships, and then
+updates the cooresponding arrays with the ships points */
     public String placeShip(String shipName, String AcrossS, String DownS, String orientation) {
```

Comment:
You've refactored a lot here. This section was especially bad, and the changes you have made a definitely paramount to cleaning up our code.